scripts/cloud-check-control/cloudAtlasCheckControl.py

```diff
@@ -0,0 +1,656 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+execute atlas-checks on an EC2 instance
+"""
+import argparse
+import logging
+import os
+import sys
+import time
+
+import boto3.ec2
+import paramiko
+import scp
+from botocore.exceptions import ClientError
+from paramiko.auth_handler import AuthenticationException
+
+
+VERSION = "0.1.0"
+ec2 = boto3.client("ec2")
+
+
+def setup_logging(default_level=logging.INFO):
+    """
+    Setup logging configuration
+    """
+    logging.basicConfig(
+        format="%(asctime)s %(levelname)-8s %(message)s",
+        level=default_level,
+        datefmt="%Y-%m-%d %H:%M:%S",
+    )
+    return logging.getLogger("CloudAtlasChecksControl")
+
+
+def finish(error_message=None, status=0):
+    """exit the process
+
+    Method to exit the Python script. It will log the given message and then exit().
+
+    :param error_message: Error message to log upon exiting the process
+    :param status: return code to exit the process with
+    """
+    if error_message:
+        logger.error(error_message)
+    else:
+        logger.critical("Done")
+    exit(status)
+
+
+class CloudAtlasChecksControl:
+    """Main Class to control atlas checks spark job on EC2"""
+
+    def __init__(
+        self,
+        timeoutMinutes=6000,
+        key="",
+        instanceId="",
+        processes=32,
+        memory=256,
+        formats="flags",
+        countries="BRA",
```

Comment:
Does it really make sense for this to have a default? Perhaps this should just be an empty string?

Reply:
Removed this default and made -c or --countries a required parameter.